examples/mnist_with_visdom.py

```diff
@@ -34,7 +34,7 @@ def forward(self, x):
         x = F.relu(self.fc1(x))
         x = F.dropout(x, training=self.training)
         x = self.fc2(x)
-        return F.log_softmax(x)
+        return F.log_softmax(x, dim=-1)
 
 
```

Comment:
Is there a reason you didn't add a handler to log validation results in this example?

Reply:
@jasonkriss do you mean no `def log_training_results(engine)` as in mnist or mnist with tensorboardX ? 
Actually, there is no particular reason except we need to create specific windows and draw lines as done in validation... :)
Yep, I will add this

Reply:
@jasonkriss I updated the example, if you want to take a look

Reply:
@vfdev-5 LGTM. Thanks!